test/test_parser.cpp

```diff
@@ -1331,7 +1332,7 @@ TEST(Parser_substitution)
     list_1.add(obj_keys[0]);
 
     util::Any args[] = {Int(2), Double(2.25), String("oe"), realm::null{}, Bool(true), Timestamp(1512130073, 505),
-                        bd0,    Float(2.33),  Int(1),       Int(3),        Int(4),     Bool(false)};
+                        bd0,    Float(2.33),  obj_keys[0],  Int(3),        Int(4),     Bool(false)};
```

Comment:
Looks like this change has a side effect which breaks another part of the test.

Reply:
@ironage it was actually the test I added that was wrong. Weird that I did not see that yesterday.